paddle/fluid/operators/save_combine_op.h

```diff
@@ -41,31 +41,40 @@ class SaveCombineOpKernel : public framework::OpKernel<T> {
 
     bool is_present = FileExists(filename);
     if (is_present && !overwrite) {
-      PADDLE_THROW("%s exists!, cannot save_combine to it when overwrite=false",
-                   filename, overwrite);
+      PADDLE_THROW(platform::errors::PreconditionNotMet(
+          "%s exists!, cannot save_combine to it when overwrite is set to "
```

Comment:
Remove `,` and capitalize the 1st letter of `cannot`.

Reply:
done, thx